src/App.jsx

```diff
@@ -0,0 +1,57 @@
+import './App.css'
+import { createContext, useEffect, useState } from 'react'
+import { Routes, Route } from 'react-router-dom'
+import FullPost from './components/FullPost'
+import Main from './components/Main'
+import Header from './components/Header'
+import NavBar from './components/NavBar'
+import Profile from './components/Profile'
+
+export const UserContext = createContext()
+export const PostsContext = createContext()
+export const NavContext = createContext()
+
+function App() {
+    const [user, setUser] = useState(null)
+
+    const [posts, setPosts] = useState([])
+
+    const [navActive, setNavActive] = useState('Home')
+
+    function getPosts() {
+        fetch('https://boolean-uk-api-server.fly.dev/angustownsley/post')
```

Comment:
Move the base URL to a const `export const API_URL = "https://boolean-uk-api-server.fly.dev/"`